torch/cuda/__init__.py

```diff
@@ -1274,6 +1274,12 @@ def kernel_impl(*args, **kwargs):
 
         return bsr_dense_mm(*args, skip_checks=True, **kwargs)
 
+    @_WrappedTritonKernel
+    def addmm_kernel_impl(*args, **kwargs):
+        from torch.sparse._triton_ops import bsr_dense_addmm
```

Comment:
I've noticed once that doing import in the hot-path is still somewhat slow, even if the module is imported already. Hope this is not the case here.